### DISCUSSION TITLE
机械加工拆图分享

### DISCUSSION THREAD

**Discussion by eqio** [peng-zhihui/ElectronBot#54] [General]
CNC加工部分
[底座上盖.pdf](https://github.com/peng-zhihui/ElectronBot/files/8246049/default.pdf)
[底座下盖.pdf](https://github.com/peng-zhihui/ElectronBot/files/8246051/default.pdf)
[cad.zip](https://github.com/peng-zhihui/ElectronBot/files/8246057/cad.zip)

**Comment by eqio** [comment]
3D打印部分
[加工件.zip](https://github.com/peng-zhihui/ElectronBot/files/8249960/default.zip)

**Reply by peiyu7921** [reply]
只能cnc吗？cnc可以用3d打印代替吗

**Reply by eqio** [reply]
验证可以吧。质感和配重不一样。

**Reply by ghost** [reply]
可以分享一下外盒的3D打印文件么？就是装小机器人的盒子。

**Comment by jinyi7016** [comment]
有没有推荐的3D打印的厂家

**Reply by yiwp9** [reply]
三维猴可以
